gulpfile.babel.js

```diff
@@ -18,7 +18,8 @@ const getEnv = () => {
   const envs = {
     local: 'http://localhost:5000',
     docker: 'http://localhost',
-    preprod: 'https://preprod-surveys.eq.ons.digital'
+    preprod: 'https://preprod-surveys.eq.ons.digital',
+    remote: 'https://' + process.env.REMOTE_ADDRESS
```

Comment:
I think it's confusing having a command line argument to choose the environment and an environment variable to set the URL. Maybe the `env` command line argument could take an environment name e.g. preprod, or a URL? If provided with a URL it would use that, else try and lookup an environment by name.